.rubocop.yml

```diff
@@ -0,0 +1,32 @@
+AllCops:
+  TargetRubyVersion: 2.5
+  NewCops: disable
+  Include:
+    - 'lib/**/*'
+  Exclude:
+    - 'spec/**/*'
+    - 'vendor/**/*'
```

Comment:
These exclusions aren't needed either, because only `lib` should be consulted per the inclusion above.

Reply:
Removed.